archivy/cli.py

```diff
@@ -38,6 +38,18 @@ def init(ctx):
     except FileNotFoundError:
         pass
 
+    # ask if to remove old users
+    try:
+        users_db = (Path(app.config['INTERNAL_DIR']) / 'db.json').resolve(strict=True)
+        remove_old_users = click.confirm("Found an existing user database. Do you want "
+                                         "to remove them?")
+        if remove_old_users:
+            users_db.unlink()
```

Comment:
This would also remove other db objects. Maybe we can just inform them there are other created users and provide a `delete-user` command?

Reply:
What other information can the db contain?